src/wagtail_vector_index/index/base.py

```diff
@@ -55,9 +65,37 @@ def query(self, query: str) -> QueryResponse[VectorIndexableType]:
             merged_context,
             query,
         ]
+
         response = self.ai_backend.chat(system_messages=[], user_messages=user_messages)
+
         return QueryResponse(response=response, sources=sources)
 
+
+    async def query_async(self, query: str) -> tuple[Callable, Iterable[VectorIndexableType]]:
```

Comment:
**suggestion:** Should we call this `aquery` for consistency with Django/Python async methods?